.travis.yml

```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - '7.0'
   - '7.2'
```

Comment:
I'd remove 7.2 to keep CI times low - testing the highest and lowest supported version should be enough

Reply:
I fixed it